src/page/Post/Post.tsx

```diff
@@ -0,0 +1,62 @@
+import React  from 'react';
+import './Post.scss'
+import { Title } from '../../components/Title';
+import { Button } from '../../components/Button'
+import { ReactComponent as Bookmark } from '../../components/Button/icon/bookmark.svg'
+import { ReactComponent as Like } from '../../components/Button/icon/like.svg'
+import { ReactComponent as Dislike } from '../../components/Button/icon/dislike.svg'
+import { ReactComponent as BookmarkTrue } from '../../components/Button/icon/Bookmark-True.svg'
+import { ReactComponent as LikeTrue} from '../../components/Button/icon/like-true.svg'
+import { ReactComponent as DislikeTrue} from '../../components/Button/icon/dislike-true.svg'
+import { useParams } from "react-router-dom"
+import { usePosts } from '../../features/posts/usePosts'
+import { Link } from 'react-router-dom';
+
+export const Post = () => {
+
+    const { posts, onLikePost, onDislikePost, onFavoritePost } = usePosts()
+
+    const { id } = useParams();
+
+    return (
+        <div className='post'>
+            <section className="post__section">                
+                <p className='post__number-post'><Link to='/' className='post__link-home'>Home </Link>| Post {id}</p>
+                <h1 className='post__title'>{posts?.[Number(id)-1].title}</h1>           
```

Comment:
Исходя из комментария выше здесь и в других свойствах должно быть как-то так
```suggestion
                <h1 className='post__title'>{title}</h1>           
```